modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -0,0 +1,589 @@
+}
+
+int removeTargetBrowserPair(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::removeTargetBrowserPair");
+    std::string id = ghoul::lua::value<std::string>(L, 1);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    TargetBrowserPair* found = module->getPair(id);
+    if (found) {
+        std::string browser = found->browserId();
+        std::string target = found->targetId();
+
+        module->removeTargetBrowserPair(id);
+
+        // Remove from engine
+        global::scriptEngine->queueScript(
+            "openspace.removeScreenSpaceRenderable('" + browser + "');",
+            scripting::ScriptEngine::RemoteScripting::Yes
+        );
+
+        global::scriptEngine->queueScript(
+            "openspace.removeScreenSpaceRenderable('" + target + "');",
+            scripting::ScriptEngine::RemoteScripting::Yes
+        );
+    }
+    
+    return 0;
+}
+
+int translateScreenSpaceRenderable(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 5, "lua::translateScreenSpaceRenderable");
+    auto [id, startX, startY, transX, transY] = 
+        ghoul::lua::values<std::string, float, float, float, float>(L);
+    
+    ScreenSpaceRenderable* renderable = global::renderEngine->screenSpaceRenderable(id);
+
+    if (renderable) {
+        renderable->translate(glm::vec2(transX, transY), glm::vec2(startX, startY));
+    }
+
+    return 0;
+}
+
+int removeSelectedImageInBrowser(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 2, "lua::removeSelectedImageInBrowser");
+    auto [id, i] = ghoul::lua::values<std::string, int>(L);
+
+    // Get browser
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    const ImageData& image = module->getWwtDataHandler()->getImage(i);
+        
+    TargetBrowserPair* pair = module->getPair(id);
+    if (pair) {
+        pair->removeSelectedImage(i);
+    }
+
+    return 0;
+}
+
+int setEquatorialAim(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 3, "lua::setEquatorialAim");
+    auto [id, ra, dec] = ghoul::lua::values<std::string, double, double>(L);
+
+    // Get module
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    TargetBrowserPair* pair = module->getPair(id);
+    if (pair) {
+        pair->setEquatorialAim(glm::dvec2(ra, dec));
+    }
+
+    return 0;
+}
+
+int setVerticalFov(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 2, "lua::setVerticalFov");
+    auto [id, vfov] = ghoul::lua::values<std::string, float>(L);
+
+    // Get module
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    TargetBrowserPair* pair = module->getPair(id);
+    if (pair) {
+        pair->setVerticalFov(vfov);
+    }
+
+    return 0;
+}
+
+int setBorderColor(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 4, "lua::setBorderColor");
+    auto [id, r, g, b] = ghoul::lua::values<std::string, int, int, int>(L);
```

Comment:
rgb could be one input instead (vec3,  or dvec3. Think w ecan only get double values using the `ghoul::lua::values` function)